plugins/bigquery/dbt/include/bigquery/macros/materializations/snapshot.sql

```diff
@@ -1,6 +1,9 @@
-{% macro bigquery__snapshot_hash_arguments(args) %}
-  to_hex(md5(concat({% for arg in args %}coalesce(cast({{ arg }} as string), ''){% if not loop.last %}, '|',{% endif %}{% endfor %})))
-{% endmacro %}
+{% macro bigquery__snapshot_hash_arguments(args) -%}
+  to_hex(md5(concat({% for arg in args %}
```

Comment:
I have similar for-loop whitespace formatting questions as I did on snowflake, here